### ISSUE TITLE
ERROR: Could not find a version that satisfies the requirement ive-tanim (from howdy)

### ISSUE THREAD

**Issue by glensc** [tanimislam/howdy#7]
I've installed via git url on macOS, Python 3.12:

```
$ ... create venv, enter venv
$ pip3 install  git+https://github.com/tanimislam/howdy.git#egg=howdy
...
INFO: pip is looking at multiple versions of howdy to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement ive-tanim (from howdy) (from versions: none)
ERROR: No matching distribution found for ive-tanim
❯ python --version
Python 3.12.3
```

doesn't seem to be present in pip at all?

```
❯ pip install ive-tanim
ERROR: Could not find a version that satisfies the requirement ive-tanim (from versions: none)
ERROR: No matching distribution found for ive-tanim
```

**Comment by glensc** [comment]
brute force guessing, all 404:
- https://pypi.org/project/ive_tanim
- https://pypi.org/project/ive-tanim

also from https://github.com/tanimislam/howdy/commit/642bf20e3159db13b7b1f10de99e342284d8edda trying previous names:
- https://pypi.org/project/iv_tanim
- https://pypi.org/project/iv-tanim

**Comment by glensc** [comment]
as a workaround, i installed [ive_tanim](https://github.com/tanimislam/ive_tanim) from git source:

```
git clone https://github.com/tanimislam/howdy
cd howdy
... init and enter venv
pip install git+https://github.com/tanimislam/ive_tanim#egginfo=ive_tanim
pip install -r requirements.txt
```

**Comment by tanimislam** [comment]
@glensc yes you are correct, ive_tanim is not on pypi. Can practically be installed either from local or from GitHub using that specified URL.

**Comment by glensc** [comment]
why it's not published in pypi? how I'm supposed to install package from pypi that depends on it?

you can use workflow from my project, it's pretty straight forward:
- https://github.com/glensc/plex-fuse/commits/0c1194e2eadcc0ddf29558cc5cf0df5126d3c05c/.github/workflows/pypi.yml

you need to adjust the version generation block, or just remove it if you modify it statically via git repo.

requirenments.txt supports URLs, but that wouldn't solve installing from with pip (it would not read your requirements.txt)
